setup.py

```diff
@@ -55,6 +55,7 @@ def find_version(*file_paths):
     author="pySerial-team",
     url="https://github.com/home-assistant-libs/pyserial-asyncio-fast",
     packages=['serial_asyncio_fast'],
+    package_data={'serial_asyncio_fast': ['py.typed']},
```

Comment:
With `pyproject.toml` configs setuptools will auto-include `py.typed` files in the package directory. It will be enough to just create the `py.typed` file.